app/src/end2endTest/resources/test-mas/claim-378-7101.json

```diff
@@ -0,0 +1,29 @@
+{
+  "dob": "1970-02-19",
+  "firstName": "Janet",
+  "lastName": "Barista",
+  "gender": "female",
+  "collectionId": 378,
+  "veteranIdentifiers": {
+    "icn": "mock1012666073V986377",
+    "ssn": "111-11-1377",
+    "veteranFileId": "9999377",
+    "edipn": "1010234-377",
+    "participantId": "88888377"
+  },
+  "claimDetail": {
+    "benefitClaimId": "1040",
```

Comment:
Now that we have mock lighhouse server I would not use the same benefit id for different end to end tests.  If nothing else to test database population is correct. Looking at this ICN's are identical to 377 as well so it looks ike copy and paste error. Looks like this is increase and the previous one presuptive. I would use different patients and make sure the  to choose a condition that does not intersects. One of them looks three years and other looks at two years I think.